### PR TITLE
Import Linux Kernel typos: letter O

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8110,10 +8124,12 @@ otherwsie->otherwise
 othewice->otherwise
 othewise->otherwise
 othewize->otherwise
+otho->otoh
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: say what?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8134,6 +8150,7 @@ ourselfs->ourselves
 ourselve->ourselves
 ourselvs->ourselves
 oustanding->outstanding
+out-of-bound->out-of-band
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: could also very well be `out-of-bounds`. I haven't encountered `out-of-band` before. Not sure about this one.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: acronym, OTOH (on the other hand)

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: oh right. heh. I guess we should keep it

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Kernel related.
Read here:
https://en.wikipedia.org/wiki/Out-of-band_management

Anyway, I'm going to remove it.

**Pull request merged**